### ISSUE TITLE
BUG V1.3.0 - Type error: Argument 1 passed to Gett\MyparcelNL\DeliverySettings\ExtraOptions::__construct()

### ISSUE THREAD

**Issue by Multipage** [myparcelnl/prestashop#95]
After you manually created the column 'extra_options' ( text) i mentioned in my previous issue the next error appears:

`Type error: Argument 1 passed to Gett\MyparcelNL\DeliverySettings\ExtraOptions::__construct() must be of the type array, null given, called in /home/XXXXXX/domains/XXXXXX/public_html/modules/myparcelnl/src/DeliverySettings/DeliverySettingsRepository.php on line 79`

**Comment by Multipage** [comment]
Quick and dirty fix for our customers:
Change:
```
        $extraOptions = [];
        if (isset($row['extra_options'])) {
            $extraOptions = json_decode($row['extra_options'], true);
        }
```

to:
```
        $extraOptions = [];
        if (!empty($row['extra_options'])) {
            $extraOptions = json_decode($row['extra_options'], true);
        }
        
```

**Comment by Multipage** [comment]
Bug still present in version 1.4.1

**Comment by RichardPerdaan** [comment]
Fixed in version [1.5.0](https://github.com/myparcelnl/prestashop/releases/tag/v1.5.0)
